pkg/table/table.go

```diff
@@ -260,6 +287,16 @@ func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case m.ShowHelp && key.Matches(msg, m.KeyMap.CloseFullHelp):
 			m.ShowHelp = false
 			return m, nil
+		case key.Matches(msg, m.KeyMap.Filter):
+			m.filterInputEnabled = !m.filterInputEnabled
```

Comment:
This means you can't have searches that contain the slash `/`

Suggest the following:

- Press slash `/` to open filter field
- Press enter to close filter field
- Press escape to cancel filter (resetting it to empty string)